### DISCUSSION TITLE
How to set the color of markdown "code"

### DISCUSSION THREAD

**Discussion by ganler** [pradyunsg/furo#173] [Questions] [answered]
Thanks for the super great work! I am wondering how can I change the [code](https://www.markdownguide.org/basic-syntax/#code) color. By default, it is white (in the dark mode), which I think is not very distinct.

For example:

![image](https://user-images.githubusercontent.com/38074777/123960534-f834a200-d9e1-11eb-965a-4a7a380c2fa4.png)

**Comment by pradyunsg** [comment, marked as answer]
There's no way to change the foreground, but there is a mechanism to set the background: override the `color-inline-code-background` variable, using the mechanism described in https://pradyunsg.me/furo/customisation/colors/#using-your-own-colors

Beyond that, this is likely resolved via the changes that have been made to the presentation of code, which now has a border.
